StudentDriver/StudentDriver/Services/WebService.cs

```diff
@@ -11,41 +11,55 @@
 using Xamarin.Forms;
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
+using StudentDriver.Helpers;
 
 namespace StudentDriver.Services
 {
     public static class WebService
     {
         private static string BaseUrl = "";
 
-        public static async Task<UserStats> GetStudentStats(int id)
+        public static async Task<string> GetAuthentiationToken(OAuthProvider.ProviderType providerType, string OAuthId)
```

Comment:
I don't fully know if this is how we are going to be doing this, I think we should follow up with @DylanWalseth on this to be sure, I could be wrong but I feel like our discussion on Monday was different then what is being done here.